allennlp/tests/modules/text_field_embedders/basic_text_field_embedder_test.py

```diff
@@ -104,26 +108,7 @@ def test_forward_runs_with_non_bijective_mapping(self):
                 }
         token_embedder(inputs)
 
-    def test_old_from_params_new_from_params(self):
```

Comment:
Especially if we're removing the deprecation warning, we should not remove this test, as we're still allowing arguments to be passed this way.  If we keep the deprecation warning as I would prefer, we should still keep this test.  We can very easily remove the displayed warning in CI by adding a `pytest.mark.filterwarnings("token_embedders")` decorator to the test.

Reply:
Good point--I can do that.

Reply:
I also added a test to make sure the keys match regardless of how they are passed in.  The old test just made sure the old way didn't throw an exception and instead logged a warning.